### ISSUE TITLE
specifying object standard class in `tripal_hq_save_submission` breaks submit

### ISSUE THREAD

**Issue by bradfordcondon** [statonlab/tripal_hq#91]
`tripal_hq_save_submission` was given a type for the object (stdObj).  but the passed object is TripalEntity.

**Comment by almasaeed2010** [comment]
Can we not type hint it? I can see I use case for this function where we pass a fake object to it.

**Comment by bradfordcondon** [comment]
also i need to specify that php 5.6 is as requirement because of the type hints

**Comment by bradfordcondon** [comment]
```
/**
 * Save the fully constructed entity as a submission.
 *
 * @param \TripalEntity $entity
 *   A tripal entity object.
 * @param int $submission_id
 *   The HQ submission ID.
 *
 * @return string
 *   The submission id, or FALSE if error.
 */
function tripal_hq_save_submission(TripalEntity $entity, int $submission_id = NULL) {
```

this appears to work

**Comment by almasaeed2010** [comment]
Yeah that should work since we currently always give it a `TripalEntity`.

**Comment by almasaeed2010** [comment]
#92 fixes this issue

**Comment by bradfordcondon** [comment]
merged #92 closing, thanks
